relevanceai/utils/transport.py

```diff
@@ -32,6 +35,28 @@ class Transport(JSONEncoderUtils, ConfigMixin):
     api_key: str
     config: Config
     logger: AbstractLogger
+    request_logger: logging.Logger
+
+    def __init__(self, request_log_filename="request.log", **kwargs):
+
+        if os.environ["DEBUG_REQUESTS"] == "1":
+            logging.basicConfig(
+                filename=request_log_filename,
+                filemode="w",
```

Comment:
This will create a new file everytime.
You should use `FileLogger` 

Please search the codebase for instances of `FileLogger` and where that is used. 

```
from relevanceai import FileLogger
with FileLogger(...): 
```